### PR TITLE
Adding Assisted Living email template

### LINKED ISSUE #268: Update Senior Housing email template
Update the, currently hardcoded, static email text with text provided by city staff.  This text will be used when `/contact/send` is triggered.

### PR DESCRIPTION
**Pull request by revjtanton**
With this change, we will send a different email to ALF's than to everyone
else.
resolves #268 

## Impacted Areas in Application

List general components of the application that this PR will affect:

- This adds some flexibility to the email blast, though this is temporary.

### CODE REVIEW

File: mail_templates/assisted_living_checkin_html.njk [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<h1>{{ emailTitle }}</h1>
+
+<p>The Baltimore City Health Department’s Division on Aging is sending out a survey to licensed Assisted Living Facilities. This survey is optional, and will be used to understand resource needs for assisted living facilities in Baltimore City. If you choose to complete this survey, all questions will need to be answered to better assist The Baltimore City Health Department in assessing areas of need or concern. All responses will be kept confidential. </p>
```
Comment (mschechter-bellese) [inline mail_templates/assisted_living_checkin_html.njk:3]: "the Baltimore City..."

File: mail_templates/assisted_living_checkin_html.njk [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                  <td style="border-radius: 2px;" bgcolor="#ED2939">
+                      <a href="{{ entityLink }}" target="_blank" style="padding: 8px 12px; border: 1px solid #ED2939;border-radius: 2px;font-family: Helvetica, Arial, sans-serif;font-size: 14px; color: #ffffff;text-decoration: none;font-weight:bold;display: inline-block;">
+                          Check In            
+                      </a>
+                  </td>
+              </tr>
+          </table>
+      </td>
+  </tr>
+</table>
```
Comment (mschechter-bellese) [inline mail_templates/assisted_living_checkin_html.njk:25]: Please add missing terminal line feed

File: mail_templates/assisted_living_checkin_html.njk [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<h1>{{ emailTitle }}</h1>
+
+<p>The Baltimore City Health Department’s Division on Aging is sending out a survey to licensed Assisted Living Facilities. This survey is optional, and will be used to understand resource needs for assisted living facilities in Baltimore City. If you choose to complete this survey, all questions will need to be answered to better assist The Baltimore City Health Department in assessing areas of need or concern. All responses will be kept confidential. </p> 
+
+<p>The Baltimore City Health Department will assess needs and work to help with resources to the best of our ability. This survey will not be used to give out supplies, but may be used to help find resources on where facilities can access those supplies if there is a shortage.</p>
```
Comment (mschechter-bellese) [inline mail_templates/assisted_living_checkin_html.njk:5]: Maybe "distribute supplies"?

File: mail_templates/assisted_living_checkin_html.njk [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+<p>We also want to provide you with additional information about support that is available to licensed facilities during COVID-19. Please join our weekly calls on Fridays from 2pm-3pm for further information on how to best protect residents and staff during this time. Information from these surveys will be used to understand topics that need to be covered in this weekly call. A link to the online meeting will be emailed out separately. </p>
```
Comment (mschechter-bellese) [inline mail_templates/assisted_living_checkin_html.njk:7]: Please change to "2:00 PM - 3:00 PM ET"

File: mail_templates/assisted_living_checkin_html.njk (hunk 2 above)
Comment (mschechter-bellese) [reply mail_templates/assisted_living_checkin_html.njk:25]: This should apply to all files

File: mail_templates/assisted_living_checkin_text.njk [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{{ emailTitle }}
+
+The Baltimore City Health Department’s Division on Aging is sending out a survey to licensed Assisted Living Facilities. This survey is optional, and will be used to understand resource needs for assisted living facilities in Baltimore City. If you choose to complete this survey, all questions will need to be answered to better assist The Baltimore City Health Department in assessing areas of need or concern. All responses will be kept confidential. 
+
+The Baltimore City Health Department will assess needs and work to help with resources to the best of our ability. This survey will not be used to give out supplies, but may be used to help find resources on where facilities can access those supplies if there is a shortage.
+
+We also want to provide you with additional information about support that is available to licensed facilities during COVID-19. Please join our weekly calls on Fridays from 2pm-3pm for further information on how to best protect residents and staff during this time. Information from these surveys will be used to understand topics that need to be covered in this weekly call. A link to the online meeting will be emailed out separately.
```
Comment (mschechter-bellese) [inline mail_templates/assisted_living_checkin_text.njk:7]: Please apply text changes from the HTML content here as well.

**Review by mschechter-bellese — CHANGES_REQUESTED** [review]
Just a couple of minor changes

File: src/routes/contact.js [hunk 6]
```diff
@@ -179,6 +179,7 @@ router.post('/send', async (req, res) => {
             name: contact.name,
```
Comment (mschechter-bellese) [inline src/routes/contact.js:179]: If I'm reading the overall flow correctly, the entity IDs and contact IDs are never used in this method. It also appears that if they are provided, the e-mail is not sent.

Am I missing something?

**Review by mschechter-bellese — CHANGES_REQUESTED** [review]
Please see comments in-line

File: src/email/index.js [hunk 7]
```diff
@@ -58,13 +58,14 @@ const sendForgotPassword = async (userEmail, resetPasswordToken) => {
 const sendContactCheckInEmail = async (info) => {
   try {
     const entityLink = `${process.env.URL}/checkin/${info.entityId}?token=${info.token}`
-    const emailTitle = `${info.entityName} Check In`
+    const emailTitle = `${info.entityName} Healthcare Rollcall Check In`
```
Comment (mschechter-bellese) [inline src/email/index.js:61]: Should this be "HealthCare RollCall"?

File: mail_templates/assisted_living_checkin_html.njk (hunk 4 above)
Comment (revjtanton) [reply mail_templates/assisted_living_checkin_html.njk:7]: The text was sent by the city. Don't want to change their copy.

File: src/routes/contact.js (hunk 6 above)
Comment (revjtanton) [reply src/routes/contact.js:179]: You're correct. This was left with a future state in mind that was put on the back burner. I'm going to revisit this now though.

File: src/email/index.js (hunk 7 above)
Comment (blakenan-bellese) [reply src/email/index.js:61]: We're sticking with the traditional (nonCamel) capitalization

**Review by blakenan-bellese — APPROVED** [review]

**Review by blakenan-bellese — APPROVED** [review]

**Pull request merged**